src/styles/utils/keyframes.scss

```diff
@@ -0,0 +1,9 @@
+@keyframes rotate {
+  from {
+    transform: translateY(-50%) rotate(0deg);
+  }
+
+  to {
+    transform: translateY(-50%) rotate(360deg);
+  }
+}
```

Comment:
since `translateY(-50%)` isn't changed here you can move this rule directly to hands
```suggestion
@keyframes rotate {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
}
```

Reply:
This suggestion breaks all)

Reply:
Not resolved. If u have any questions - feel free to ask in fe_chat